### DISCUSSION TITLE
Unpoly 2.7.2 released

### DISCUSSION THREAD

**Discussion by triskweline** [unpoly/unpoly#458] [Announcements]
This is a maintenance release to bridge the time until [Unpoly 3](https://github.com/unpoly/unpoly/discussions/407) is completed. This release includes the following changes:

- When pushing a history state, don't mutate the previous history state.
- Fix a bug where the `[up-animation]` attribute could not be used to control a new overlay's opening animation.
- The NPM package is now 800KB smaller.

This is the last release with support for Internet Explorer 11. Future releases will support Chrome, Firefox, Edge and the last two majors of Safari.

**Comment by alnuaimi94** [comment]
Is unpoly support resumebility like qwik framework or hydration like others front end frameworks?!

**Reply by triskweline** [reply]
> Is unpoly support resumebility like qwik framework or hydration like others front end frameworks?!

Unpoly apps render HTML on the server and enhance them on the client. There is no double rendering that needs to be fixed by resumability.
